src/utl/test/test_error.py

```diff
@@ -0,0 +1,6 @@
+import utl
+
+try:
+    utl.error(utl.CTS, 99, "Arbitrary CTS error message")
+except Exception as inst:
+    print(inst.args[0])
```

Comment:
You get a stack trace. The try catch here is to be sure you don't get that so that when a tcl test uses catch, we can use try catch in Python so the log files will match